src/app/_components/PageSection.tsx

```diff
@@ -1,26 +1,56 @@
+import Image from 'next/image'
+
+import {
+  type DescriptionTextType,
+  DescriptionText,
+} from '@/components/DescriptionText'
 import { Heading } from '@/components/Heading'
 import { SectionDivider } from '@/components/SectionDivider'
 
+import { ImageProps } from '@/types/sharedProps/imageType'
+
 type PageSectionProps = {
   kicker: string
   title: string
-  description?: string
+  description?: DescriptionTextType
+  image?: ImageProps
   children?: React.ReactNode
 }
 
+const containerClasses = (hasImage: Boolean) => {
+  return hasImage
+    ? 'grid grid-cols-1 gap-6 sm:grid-cols-2'
+    : 'mb-6 max-w-[60ch]'
```

Comment:
I believe we settled on `max-w-75` - We should standardize this by adding it to the config. Something like:

```
module.exports = {
  theme: {
    extend: {
      maxWidth: {
        readable: '60ch'
      }
    }
  }
}
```

This way, we can use `max-w-readable` allover.